plugins/plugin-chart-pivot-table/src/PivotTableChart.tsx

```diff
@@ -44,10 +44,16 @@ const Styles = styled.div<PivotTableStylesProps>`
       margin: ${margin}px;
       height: ${height - margin * 2}px;
       width: ${width - margin * 2}px;
-      overflow-y: scroll;
  `}
 `;
 
+const PivotTableWrapper = styled.div`
+  height: 100%;
+  max-width: 100%;
+  overflow: auto;
+  width: min-content;
```

Comment:
Can we replace `max-width: 100%` and `width: min-content` with `max-width: fit-content`? 
Using `width: min-content` causes some unnecessary wrapping in the table, whereas `max-width: fit-content` doesn't affect the table, but sets the containers width to the child's width.
According to https://caniuse.com/?search=fit-content it's widely supported

https://user-images.githubusercontent.com/15073128/129593475-c849e84c-63a8-4e75-a953-377dbd55839e.mov

Reply:
Hey @kgabryje that was a very nice catch! Thanks for that. I changed it.